lib/gui/modules/analytics.js

```diff
@@ -130,7 +68,7 @@ analytics.service('AnalyticsService', function($log, $window, $mixpanel, Setting
    * @public
    *
    * @description
-   * This function sends the debug message to TrackJS and Mixpanel.
+   * This function sends the debug message to Mixpanel and Sentry.
```

Comment:
I think we can isolate Etcher from the services being used. Something like "THis function sends the debug message to error reporting services" should suffice.